src/components/common/Error/NotFound.tsx

```diff
@@ -0,0 +1,18 @@
+import Header from "../../Header/Header";
+import Error from "./Error";
+
+import styles from "./error.module.css";
+
+function NotFound() {
+  return (
+    <div>
+      <Header />
```

Comment:
For non-existed route you shouldn't show your Header component, well-known practice is to show error message on 404 page and redirect user to the main page